src/Mod/Path/PathScripts/PathDressupLeadInOut.py

```diff
@@ -647,97 +652,30 @@ def accept(self):
-        self.form.chkRapidPlunge.setChecked(self.obj.RapidPlunge)
-        self.form.chkLayers.setChecked(self.obj.IncludeLayers)
-        self.form.chkKeepToolDown.setChecked(self.obj.KeepToolDown)
-        self.form.chkUseCRC.setChecked(self.obj.UseMachineCRC)
-
-        self.form.dsbLen.setValue(self.obj.Length)
-
-        self.form.dsbExtendIn.setValue(self.obj.ExtendLeadIn)
-        # self.form.dsbExtendIn.setEnabled(self.obj.LeadIn)
-
-        self.form.dsbExtendOut.setValue(self.obj.ExtendLeadOut)
-        # self.form.dsbExtendOut.setEnabled(self.obj.LeadOut)
-
-        self.form.cboStyleIn.setCurrentIndex(
-            self.obj.getEnumerationsOfProperty("StyleOn").index(self.obj.StyleOn)
-        )
-        # self.form.cboStyleIn.setEnabled(self.obj.LeadIn)
-
-        self.form.cboStyleOut.setCurrentIndex(
-            self.obj.getEnumerationsOfProperty("StyleOff").index(self.obj.StyleOff)
-        )
-        # self.form.cboStyleOut.setEnabled(self.obj.LeadOut)
-
-        self.form.cboRadius.setCurrentIndex(
-            self.obj.getEnumerationsOfProperty("RadiusCenter").index(
-                self.obj.RadiusCenter
-            )
-        )
-
     def updateModel(self):
         self.getFields()
+        self.obj.Proxy.execute(self.obj)
         FreeCAD.ActiveDocument.recompute()
 
-    def setFields(self):
-        self.updateUI()
-
     def open(self):
         pass
 
     def setupUi(self):
-        self.form.cboStyleIn.clear()
-        self.form.cboStyleIn.addItems(
-            [
-                translate("Path_DressupLeadInOut", itm)
-                for itm in self.obj.getEnumerationsOfProperty("StyleOn")
-            ]
-        )
-        self.form.cboStyleOut.clear()
-        self.form.cboStyleOut.addItems(
-            [
-                translate("Path_DressupLeadInOut", itm)
-                for itm in self.obj.getEnumerationsOfProperty("StyleOff")
-            ]
-        )
-        self.form.cboRadius.clear()
-        self.form.cboRadius.addItems(
-            [
-                translate("Path_DressupLeadInOut", itm)
-                for itm in self.obj.getEnumerationsOfProperty("RadiusCenter")
-            ]
-        )
+        self.connectWidget("LeadIn", self.form.chkLeadIn)
+        self.connectWidget("LeadOut", self.form.chkLeadOut)
+        self.connectWidget("Length", self.form.dsbLen)
+        self.connectWidget("ExtendLeadIn", self.form.dsbExtendIn)
+        self.connectWidget("ExtendLeadOut", self.form.dsbExtendOut)
+        self.connectWidget("StyleOn", self.form.cboStyleIn)
+        self.connectWidget("StyleOff", self.form.cboStyleOut)
+        self.connectWidget("RadiusCenter", self.form.cboRadius)
+        self.connectWidget("RapidPlunge", self.form.chkRapidPlunge)
+        self.connectWidget("IncludeLayers", self.form.chkLayers)
+        self.connectWidget("KeepToolDown", self.form.chkKeepToolDown)
+        self.connectWidget("UseMachineCRC", self.form.chkUseCRC)
```

Comment:
@sliptonic This shows how simple it is to connect together widgets and properties for these super simple UIs, all the mess is hidden in the mixin class.